### ISSUE TITLE
Some tools with robots do not work

### ISSUE THREAD

**Issue by soulcat1** [AlexTheGre4t/MCTNGIssues#34]
I found certain tool using robots do not work because the tools are useless and they will not use tinkers tools.  The lumber jack I found works with regular axe, however the Farmer does not with regular Hoes.

**Comment by AlexTheGre4t** [comment]
Ugh. Dammit why does these things become an issue :/

I'm changing the TiC system a lot soon anyway. I'm considering re-enabling some vanilla tools but have them slightly nerfed to make Tinkers more desirable and to make people want to use them more.

**Comment by Xariyl** [comment]
Considering the upgrades and potential power of the Tinker's tools, I don't know that vanilla tools have much to offer in comparison. TiC tools and weapons offer big advantages.

For the sake of making recipes easier/less buggy with MT, compatibility with robots and other mods, I think we should strongly consider reenabling them.

We can always MT them to use TiC parts in the recipe but produce vanilla results. Gives us cross mod compatibility while remaining integrated with the TiC path.

**Comment by AlexTheGre4t** [comment]
Yes. Ive been thinking about this and they will be re enabled again. Though the idea of using TiC parts is a very good idea.
